tests/ieee802154_hal/main.c

```diff
@@ -37,7 +37,7 @@
 #include "test_utils/expect.h"
 #include "xtimer.h"
 
-#define SYMBOL_TIME (256U) /**< 16 us */
+#define SYMBOL_TIME (1024U) /**< 16 us */
```

Comment:
since this application should also work for O-QPSK radios, we will need to calculate this on demand. I plan to propose a PR with a mechanism to do that (since we also need it for other SubGHz radios).